source/_integrations/zwave_js.markdown

```diff
@@ -100,11 +110,14 @@ Valid code slots are between 1-254.
 
 This event is fired whenever a [notification](https://zwave-js.github.io/node-zwave-js/#/api/node?id=quotnotificationquot) or [value notification](https://zwave-js.github.io/node-zwave-js/#/api/node?id=quotvalue-notificationquot) event is received.
 
+Events are issued for example when you press some button on a Z-Wave remote (aka Central Scene support) or when some other stateless value is being signalled by your device. You can test what events come in by using the developer tools in Home Assistant and listen for "zwave_js_event". Once you know what the event data looks like, you can use this to create automations. In the future we'd like to provide you with some autogenerated device automations for these kind of events.
```

Comment:
I know this is new but Central Scenes are mentioned below and yet people continue to ask questions. We may want to make Scenes a section level header so it's easier to find

Reply:
changed the headers to make it (even) more clear